Autopilot/PathManager/Inc/CommsWithTelemetry.hpp

```diff
@@ -11,19 +11,19 @@
 /**
  * Initiate communication with telemetry.
  */
-void CommWithTelemInit();
+void CommFromPMToTMInit();
 
 /**
  * Send path manager data to telemetry.
  * @param[in]   data   Pointer to struct with data.
  */
-void SendPathData(Telemetry_POGI_t *data);
+void SendFromPMToTM(Telemetry_POGI_t *commands);
 
 /**
  * Retrieve commands from the path manager.
  * @param[out]   commands   Pointer to struct to recieve commands.
  * @return True if commands were recieved.
  */
-bool GetTelemetryCommands(Telemetry_PIGO_t *commands);
+bool GetFromPMToTM(Telemetry_POGI_t *commands);
```

Comment:
In the other interface, this is organized differently since this function is called by telemetry, not path manager. To be more consistent, this function declaration would be swapped with "GetFromTMToPM(Telemetry_PIGO_t *commands)", which is called by path manager. The current way works, but forces telemetry to include this header instead of keeping them isolated.

Reply:
Oh okay. So I will just do an ol' switcheroo

Reply:
Turns out switching them causes some odd linking errors. I'll keep them as is